packages/cli/src/utils/miscellaneous.ts

```diff
@@ -662,3 +659,23 @@ export function checkForDeprecatedOptions<T>(argv: T, deprecatedOptions: Array<k
     }
   }
 }
+
+export function notifyAboutIncompatibleConfigOptions(openapi: Record<string, unknown> | undefined) {
+  if (isPlainObject(openapi)) {
+    const propertiesSet = Object.keys(openapi);
+    const deprecatedSet = Object.keys(deprecatedRefDocsSchema.properties);
+    const intersection = propertiesSet.filter((prop) => deprecatedSet.includes(prop));
+    if (intersection.length > 0) {
+      process.stderr.write(
+        yellow(
+          `\n${pluralize('Property', intersection.length)} ${gray(
+            intersection.map((prop) => `'${prop}'`).join(', ')
+          )} ${pluralize(
+            'is',
+            intersection.length
+          )} specific to Redoc v1 or Reference Docs and not supported by Redoc v2.\n\n`
```

Comment:
This is the message that users will see if they use the old Reference Docs options in theme.openapi.

Reply:
The current open source version of Redoc is 2.x, and I think that does still support some of those fields. How about:

" ... only used in API Reference Docs and Redoc version 2.x or earlier."

I suggested that with the major divergence (API Reference Docs forked years ago and was sort of a v3 under other branding), we make the realm version of Redoc a v4 release, but I'm not sure I've won my case. Either way, it'll have to be at least one major version up from the current 2.x series so I think this message will work whichever way we go on that.

Reply:
Oh, indeed. Thanks for the correction. Will push an update.

Reply:
@lornajane updated